kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -299,7 +299,7 @@ def exec_prove(
     auto_abstract_gas: bool = False,
     fail_fast: bool = False,
     always_check_subsumption: bool = True,
-    fast_check_subsumption: bool = True,
+    fast_check_subsumption: bool = False,
     fallback_on: Iterable[FallbackReason] | None = None,
```

Comment:
@palinatolmach @tothtamas28 @anvacaru let's make sure when we switch a CLI option like this one, we _also_ switch the default in `cli.py` as well to match it https://github.com/runtimeverification/evm-semantics/blob/9e88278f72aebf74d94b928654b6364086bf165a/kevm-pyk/src/kevm_pyk/cli.py#L70

Reply:
Possibly related: https://github.com/runtimeverification/kontrol/pull/298#pullrequestreview-1838295387

Reply:
Right, thanks! I created an issue so that I don't forget to address it within the next few days: https://github.com/runtimeverification/evm-semantics/issues/2295, and I'll check with @nwatson22 on whether we might incorporate https://github.com/runtimeverification/kontrol/pull/298#pullrequestreview-1838295387 in fixing that as well.